### PR TITLE
Node 8 is now the required version of node

### PR DESCRIPTION
**Pull request by davidkel**
This addresses issue #2100 
We now have made Node 8 the required version for composer as it has now gone LTS. This brings much wanted language improvements for non TP javascript such as async/await.

Signed-off-by: Dave Kelsey d_kelsey@uk.ibm.com

### CODE REVIEW

**Comment by davidkel** [comment]
This doesn't mean that node 8 syntax can be used in our code base yet. We need to look at what is required for code that runs in the browser and also need to wait for us to drop support for duktape and move the the node.js support in the upcoming fabric 1.1 release.

File: packages/composer-cli/docker/Dockerfile.ppc64le [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ppc64le/node:6.11
+FROM ppc64le/node:8-8-alpine
```
Comment (sstone1) [inline packages/composer-cli/docker/Dockerfile.ppc64le]: Typo for `8-8-` here?

File: packages/composer-playground/docker/Dockerfile.ppc64le [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ppc64le/node:6.11
+FROM ppc64le/node:8-8-alpine
```
Comment (sstone1) [inline packages/composer-playground/docker/Dockerfile.ppc64le]: Typo for `8-8-` here?

File: packages/composer-rest-server/docker/Dockerfile.ppc64le [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM ppc64le/node:6.11
+FROM ppc64le/node:8-8-alpine
```
Comment (sstone1) [inline packages/composer-rest-server/docker/Dockerfile.ppc64le]: Typo for `8-8-` here?

**Comment by davidkel** [comment]
Rats, how did I manage that..... :-)

**Pull request merged**